addon/components/bs-modal.js

```diff
@@ -193,6 +193,20 @@ export default class Modal extends Component {
   @arg
   scrollable = false;
 
+    /**
+   *  Allows adding fullscreen mode for modals. It will
+   *  apply the `modal-fullscreen' class when using 'true' and
+   *  `modal-fullscreen-[x]-down` class when using BS breakpoints
+   *   ([x] = 'sm', 'md', 'lg', 'xl', 'xxl').
+   *  
+   * @property fullscreen
+   * @type {string}
```

Comment:
Reading the description I think the type is either `boolean` or `string`.

```suggestion
   * @type {(Boolean|String)}
```

Reply:
Fixed